GDJS/Runtime/pixi-renderers/spriteruntimeobject-pixi-renderer.ts

```diff
@@ -54,22 +54,21 @@ namespace gdjs {
      * Update the internal PIXI.Sprite position, angle...
      */
     _updatePIXISprite() {
-      if (this._object._animationFrame !== null) {
+      const spriteTextureFrame = this._sprite.texture.frame;
```

Comment:
I think we've never observed the problem at runtime, and runtime should never have the problem because textures must always be loaded beforehand... I would not include this change then. cc @D8H

Reply:
I agree